snactor/executors/default.py

```diff
@@ -42,32 +42,32 @@ def __init__(self, definition):
         self.log = logging.getLogger(self.definition.name).getChild(self.__class__.__name__)
 
     def handle_stdin(self, input_data):
-        self.log.debug("handle_stdin()")
+        self.log.debug("handle_stdin(%s) of '%s'", input_data, self.definition.name)
```

Comment:
This can be solved by setting the right format for the logger to include the logger name, then this is all unnecessarily verbose in the log messages.

Look at line 42 there is the definition.name passed to getLogger which in turn will create a logger with the name of the actor - It just has to be incorporated in the format string in the logger configuation

Reply:
Instead you would want just to change https://github.com/leapp-to/snactor/blob/master/tools/snactor_runner#L67

format='%(levelname)s: %(message)s' => format='%(levelname)s [%(name)s]: %(message)s'

Reply:
Here are more of those options if you also want include the time or so: https://docs.python.org/2/library/logging.html#logging.LogRecord

Reply:
Thx for the tip. I'm checking it out.